### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by alberttwong**
added howto to get the APB into the service catalog.

### CODE REVIEW

**Comment by alberttwong** [comment]
@jankleinert @jorgemoralespou  Can you help?  I don't understand why the check failed.  It looks like the password isn't being passed into docker login.

File: README.md [hunk 1]
```diff
@@ -2,6 +2,31 @@
 
 Workshop content designed to be used by the [Workshopper tool](https://github.com/osevg/workshopper).
 
+## Install the ASB into your Service Catalog
+
+An [APB](https://hub.docker.com/r/openshiftapb/cloudnative-workshop-apb) is provided for
```
Comment (jorgemoralespou) [inline README.md]: This references another APB. This should be https://hub.docker.com/r/openshiftapb/starter-workshop-apb/

File: README.md [hunk 2]
```diff
@@ -2,6 +2,31 @@
 
 Workshop content designed to be used by the [Workshopper tool](https://github.com/osevg/workshopper).
 
+## Install the ASB into your Service Catalog
```
Comment (jorgemoralespou) [inline README.md]: Install the ASB? Isn't these instructions to install the APB? Reading your instructions seems that you expect the ASB (Ansible Service Broker???) to be there

**Review by jorgemoralespou — CHANGES_REQUESTED** [review]
There's 2 minor things that need to be changed. Also, travis is not configured to push the images the PR generates to dockerhub, hence the error you see. Once merged, it will work. :-D

File: README.md (hunk 2 above)
Comment (alberttwong) [reply README.md]: Ooops.  I think I misspelled.

File: README.md (hunk 1 above)
Comment (alberttwong) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (alberttwong) [reply README.md]: fixed

**Pull request merged**